metaflow/metadata_provider/metadata.py

```diff
@@ -630,6 +630,20 @@ def _get_system_info_as_dict(self):
             sys_info["r_version"] = env["r_version_code"]
         return sys_info
 
+    def _get_git_info_as_dict(self):
+        git_info = {}
+        env = self._environment.get_environment_info()
+        for key in [
+            "script",
```

Comment:
this is already in the dict for environment. We should probably not record script. It's not emitted anyways.